src/components/Navbar/Navbar.jsx

```diff
@@ -1,13 +1,15 @@
 import "../Navbar/navbar.css";
 import { Link } from "react-router-dom";
 import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
```

Comment:
You can take a look in to absolute path to avoid ../../
**You can see from here**  [https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i](url)